examples/discovery-search-app/src/setupProxy.js

```diff
@@ -8,18 +8,18 @@ const cors = require('cors');
 const bodyParser = require('body-parser');
 
 if (serverEnv.error) {
-  console.warn(serverEnv.error);
+  throw new Error(serverEnv.error);
```

Comment:
Noticed that the travis builds are failing here, since they must not have a `.server-env` file. Not sure what the best solution is.

Reply:
I would change the cypress tests to not start the server